junit5-extension/src/main/java/io/kroxylicious/testing/kafka/clients/CloseableProducer.java

```diff
@@ -23,8 +23,23 @@
 import org.apache.kafka.common.TopicPartition;
 import org.apache.kafka.common.errors.ProducerFencedException;
 
+/**
+ * The type Closeable producer.
```

Comment:
```suggestion
 * Provides a simple wrapper around a Kafka Producer to redirect `close()` so that it has a sensible timeout and can thus be safely used in a try-with-resources block. All other methods delegate to the wrapped Producer 
```